README.md

```diff
@@ -1,6 +1,39 @@
 # Gnosis Protocol Challenge payout
 
-Just short lived repo for generating a JSON file for the payment of Gnosis Protocol Challenge:
-https://explore.duneanalytics.com/public/dashboards/aPV5YhphBMyc6I0aRKdGwu4RDMI9xqZ30qcn8FTL
+Ad-hoc script that transforms a leader-board in Dune into payment file consumed by the **multisend** script in
+[https://github.com/gnosis/dex-liquidity-provision](https://github.com/gnosis/dex-liquidity-provision/)
 
-> Probably, this code can be adapted an migrated to other project or Gnosis Github
+> **NOTE**:
+> Just short lived repo for generating a JSON file for the payment of Gnosis Protocol Challenge:
+> https://explore.duneanalytics.com/public/dashboards/aPV5YhphBMyc6I0aRKdGwu4RDMI9xqZ30qcn8FTL
+>
+> Probably, this code can be in the future generalized and adapted for other possible uses. For now. Let's keep it simple 🏄‍♂️
+
+## Setup dependencies
+
+```bash
+yarn install
+```
+
+## About the input file
+
+Generated automatically by exporting the leader board in [https://explore.duneanalytics.com/public/dashboards/aPV5YhphBMyc6I0aRKdGwu4RDMI9xqZ30qcn8FTL](https://explore.duneanalytics.com/public/dashboards/aPV5YhphBMyc6I0aRKdGwu4RDMI9xqZ30qcn8FTL)
```

Comment:
Could we include the query here too or is this too much unnecessary duplication?

Reply:
I think it's a bit out of scope that. Also could be easily outdated. But if we really want to, I don't mind to add it.